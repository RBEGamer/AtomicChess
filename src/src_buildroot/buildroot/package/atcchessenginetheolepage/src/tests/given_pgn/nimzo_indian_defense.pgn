[Event "?"]
[Site "?"]
[Date "????.??.??"]
[Round "?"]
[White "?"]
[Black "?"]
[Result "*"]
[opening "nimzo indian defense"]

1. d2-d4 Ng8-f6 2. c2-c4 e7-e6 3. Nb1-c3 Bf8-b4 *

