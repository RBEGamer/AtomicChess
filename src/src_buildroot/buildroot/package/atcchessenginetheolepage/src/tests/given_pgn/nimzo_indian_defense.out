[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 3
